test/extensions/quic_listeners/quiche/integration/quic_http_integration_test.cc

```diff
@@ -391,9 +393,13 @@ TEST_P(QuicHttpIntegrationTest, TestDelayedConnectionTeardownTimeoutTrigger) {
             1);
 }
 
+// Only run on platforms that support BPF
+#if defined(SO_ATTACH_REUSEPORT_CBPF) && defined(__linux__)
```

Comment:
why not just `if defined(SO_ATTACH_REUSEPORT_CBPF)`?

Reply:
we matched what was in here: https://github.com/envoyproxy/envoy/blob/9025fba2cc64c9b7027a02241590a8e6ac3fbd87/source/extensions/quic_listeners/quiche/active_quic_listener.cc#L229

looks like the test has undergone this change already looking back at the blame

Reply:
see this diff: https://github.com/envoyproxy/envoy/commit/f8c3acc48ec287bc239f8e35709efe74d93ddb41#diff-5872f75ef2969ee19f048c4dfc487bb7acc649021940a8acca7a4921b2d5c910

not sure why exactly the ifdef was taken out for platforms that do not support BPF

Reply:
originally added here: https://github.com/envoyproxy/envoy/commit/e176b307bd2ab475574fc7c4b0f92e8575129573#diff-5872f75ef2969ee19f048c4dfc487bb7acc649021940a8acca7a4921b2d5c910R233